package.json

```diff
@@ -6,14 +6,24 @@
   "module": "dist/esm/pushin.js",
   "main": "dist/umd/pushin.js",
   "typings": "dist/esm/index.d.ts",
+  "unpkg": "dist/umd/pushin.min.js",
   "exports": {
     "./pushin.css": {
       "style": "./dist/pushin.css"
     }
   },
+  "files": [
+    "dist",
+    "LICENSE",
+    "README.md"
+  ],
+  "engines": {
+    "node": ">= 16.0.0"
```

Comment:
I've set it to 16 since this is the latest LTS version.

Reply:
that works for me